modin/backends/pandas/query_compiler.py

```diff
@@ -1313,6 +1299,28 @@ def is_tree_like_or_1d(calc_index, valid_index):
         result = self.__constructor__(new_modin_frame)
 
         def compute_index(index, columns, consider_index=True, consider_columns=True):
+            """
+            Compute new index for the unstacked frame.
+
+            Parameters
+            ----------
+            index : pandas.Index
+                Index of the originam frame.
+            columns : pandas.Index
+                Columns of the original frame.
+            consider_index : bool, default: True
+                Whether original index contains duplicated values.
+                If True all duplicates will be droped.
+            consider_columns : bool, default: True
+                Whether original columns contains duplicated values.
+                If True all duplicates will be droped.
```

Comment:
are you sure these arguments are doing what you're writing here?.. it does not correlate with their naming IMHO...

Reply:
When you're looking into the code, parameters seem to be used as documented. @prutskov since you were the author of the PR (#2012) that added this can you recall what does this parameter means?